apis/ec2/v1beta1/routetable_types.go

```diff
@@ -171,11 +175,16 @@ type RouteTableParameters struct {
 	// Region is the region you'd like your VPC to be created in.
 	Region string `json:"region"`
 
+	// Indicates whether we reconcile inline routes
+	// +optional
+	IgnoreRoutes *bool `json:"ignoreRoutes,omitempty"`
+
 	// The associations between the route table and one or more subnets.
 	Associations []Association `json:"associations"`
 
-	// the routes in the route table
-	Routes []RouteBeta `json:"routes"`
+	// inline routes in the route table
+	// +optional
+	Routes []RouteBeta `json:"routes,omitempty"`
```

Comment:
Can you mark this field and the type as deprecated?

Reply:
what did we need to do for deprecation ?

Reply:
@muvaf  did we have an example for deprecation ?

Reply:
@haarchri @muvaf Is this the way https://github.com/crossplane/provider-aws/blob/master/apis/cache/v1beta1/replication_group_types.go#L355 ?

Reply:
Standard Golang deprecation notice: https://github.com/golang/go/wiki/Deprecated